frontend/src/App.tsx

```diff
@@ -112,88 +118,91 @@ export default function App() {
 
     function deleteEntry(id: string) {
         toast.promise(
-        axios.delete(`/api/diary/${id}`)
-            .then(() => {
-                setEntries(entries.filter(entry => entry.id !== id))
-            })
-            .catch(error => console.log(error)),
-                {
-                    pending: 'Deleting is pending',
-                    success: 'Entry deleted! 👌',
-                    error: 'Deleting rejected 🤯'
-                }
-            );
+            axios.delete(`/api/diary/${id}`)
+                .then(() => {
+                    setEntries(entries.filter(entry => entry.id !== id));
+                })
+                .catch(error => console.log(error)),
+            {
+                pending: 'Deleting is pending',
+                success: 'Entry deleted! 👌',
+                error: 'Deleting rejected 🤯'
+            }
+        );
     }
 
     function handleDescriptionChange(id: string, newDescription: string) {
         setEntries(entries.map(entry =>
-            entry.id === id ? {...entry, description: newDescription} : entry
+            entry.id === id ? { ...entry, description: newDescription } : entry
         ));
     }
 
     function handleStatusChange(id: string, newStatus: "LESS_THAN_SIX_THOUSAND_STEPS" | "SIX_THOUSAND_STEPS" | "EIGHT_THOUSAND_STEPS" | "TEN_THOUSAND_STEPS" | "MORE_THAN_TEN_THOUSAND_STEPS") {
         setEntries(entries.map(entry =>
-            entry.id === id ? {...entry, status: newStatus} : entry
+            entry.id === id ? { ...entry, status: newStatus } : entry
         ));
     }
 
     function deleteImage(id: string) {
         toast.promise(
-        axios.delete(`/api/diary/${id}/image`)
-            .then(() => {
-                setEntries(entries.map(entry =>
-                    entry.id === id ? {...entry, imageUrl: undefined} : entry
-                ));
-            })
-            .catch(error => console.log(error)),
-        {
-            pending: 'Deleting is pending',
+            axios.delete(`/api/diary/${id}/image`)
+                .then(() => {
+                    setEntries(entries.map(entry =>
+                        entry.id === id ? { ...entry, imageUrl: undefined } : entry
+                    ));
+                })
+                .catch(error => console.log(error)),
+            {
+                pending: 'Deleting is pending',
                 success: 'Image deleted! 👌',
-            error: 'Deleting rejected 🤯'
-        }
-    );
+                error: 'Deleting rejected 🤯'
+            }
+        );
     }
 
     useEffect(() => {
-        fetchData()
-        getMe()
+        fetchData();
+        getMe();
     }, []);
 
-    return(
+    return (
         <>
+            <ThemeProvider theme={theme === 'light' ? lightTheme : darkTheme}>
             <GlobalStyles />
```

Comment:
Sicne you wrap everything in the Themeprovider, the fragment (empty tags) with just a single child is redundant